Client.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <iostream>
+#include <string>
+#include <thread> 
+#include "Pdu.hpp"
+#include <cstdint>
+#include <unordered_map>
+#pragma comment(lib,"ws2_32.lib")
+
+
+class Client
+{
+private:
+	WSADATA _ws;
+	SOCKET _client_fd;
+	struct sockaddr_in _clientAddr;
+	std::string _color;
+	std::string _username;
+	const std::string RESET_COLOR = "\033[0m";
+	static constexpr const char* IP = "127.0.0.1";
+	static constexpr int PORT = 8080;
+	static constexpr int BUFFERSIZE = 1058;
+	static constexpr int NAME_SIZE = 30;
+	
+public:
```

Comment:
even though you don't use a destructor here I would like to see it 
even like this `~Client() = default` and read about default functions if you want to use it like this